pkgs/jnigen/lib/src/bindings/renamer.dart

```diff
@@ -94,20 +90,21 @@ const Map<String, int> _definedSyms = {
   'release': 1,
   'releasedBy': 1,
   'jClass': 1,
+  'type': 1,
 };
 
-/// Appends 0 to [name] if [name] is a keyword.
+/// Appends $ to [name] if [name] is a keyword.
```

Comment:
Is it a Dart or Java keywords. (Very related, not exactly the same.)

Reply:
Yes. I plan to reduce them. https://github.com/dart-lang/native/issues/664